jest/private/jest_test.bzl

```diff
@@ -77,6 +78,11 @@ def _impl(ctx):
         },
     )
 
+    # Unwind the chdir argument to adapt the path arguments
+    unwind_chdir_prefix = ""
+    if ctx.attr.chdir:
+        unwind_chdir_prefix = "/".join([".."] * len(ctx.attr.chdir.split("/"))) + "/"
```

Comment:
We've written this `../../..` generating code so many times; lets upstream it to a bazel-lib utility function one day :)

Reply:
Yeah that's what I was thinking, and when we do we can think about maybe handling `.`, `..` etc